src/pymor/algorithms/hapod.py

```diff
@@ -21,6 +23,9 @@ class Tree(BasicObject):
     def children(self, node):
         pass
 
+    def after(self, node):
```

Comment:
maybe add some docs. on first read I am unsure about the direction of the dependency